### ISSUE TITLE
Fun Run - Put wording across Scoresheet

### ISSUE THREAD

**Issue by jeanniebadger** [jpschewe/fll-sw#783]
The Current thought is to NOT show the score for FUN RUN up on the screen.  It will be given back to teams with a kind note from the REFS.  

So add "FUN RUN" across page similar to PRACTICE.

Priority: LOW - we can do this without having to change the software.  If it can be modified easily, it would be nice.

**Comment by jpschewe** [comment]
@jeanniebadger  is the "FUN RUN" all runs after the regular match play when not running head to head?

**Comment by jeanniebadger** [comment]
Yes, it would be the scheduled 4th run (or 5th round if counting practice as a run)

**Comment by jpschewe** [comment]
Ok, I'll need to think about if there is any reason to have runs beyond the regular match play rounds other than head to head and fun runs.

**Comment by jpschewe** [comment]
With the recent changes to do everything electronically, this isn't needed.
